packages/surveys/src/components/questions/cal-question.test.tsx

```
@@ -1,93 +1,134 @@
+}));
+
+vi.mock("@/components/buttons/submit-button", () => ({
+  SubmitButton: ({ buttonLabel }: { buttonLabel: string }) => (
+    <button type="submit" data-testid="submit-button">
+      {buttonLabel}
+    </button>
+  ),
+}));
+
+vi.mock("@/components/buttons/back-button", () => ({
+  BackButton: () => <button data-testid="back-button">Back</button>,
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)